appinventor/appengine/war/tutorials/Tutorial_TalkToMe.js

```diff
@@ -0,0 +1,135 @@
+/* Step Object */
+// step = function(text, validate, url){
```

Comment:
This is an example of how each step in a walkthrough tutorial should be structured in the code. Should it be placed elsewhere? I'm hesistant to delete it.